### PR TITLE
Fixes: P5-4 - Problem statement 

### PR DESCRIPTION
**Pull request by C0DK**
Mangler korrekturlæsning - hvilket jeg gerne vil have en hånd med. også et generelt take på indhold.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e88a4b0c1e0c5db2728fd196f127dac4a55e379e
  * Commits must be in the following format: **%{type}: %{description}**

 * Commit: ae280af778db7ecddcf186e4f391d4393c0a3317
  * Commits must be in the following format: **%{type}: %{description}**

Guidelines are available at https://github.com/aau-hcl-p5/main

---
This message was auto-generated by https://gitcop.com

**Comment by C0DK** [comment]
jeg squasher lige og fixer commits ogog derefter team city :P

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e88a4b0c1e0c5db2728fd196f127dac4a55e379e
  * Commits must be in the following format: **%{type}: %{description}**

 * Commit: ae280af778db7ecddcf186e4f391d4393c0a3317
  * Commits must be in the following format: **%{type}: %{description}**

Guidelines are available at https://github.com/aau-hcl-p5/main

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: ab45d0166a803f4998842f75150ee4116bda7dfd
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/aau-hcl-p5/main

---
This message was auto-generated by https://gitcop.com

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 1]
```diff
@@ -1,9 +1,56 @@
...
+% really where our solution originates, based on a real world problem,
+% even though our project is based on a "fun" project, it should still be possible to abstract it into real world applications
+
 \chapter{Introduction}
-some text
 
+Tracking airborne objects can be quite difficult.
+This difficulty has inspired different sports and hobbies, including the shooting of clay pigeons.
+Hitting a target requires the tracking of it, it's velocity and direction, as well as the calculation or prediction of its future trajectory.
+The trajectory can be impacted by number of different features, including gravity, wind, initial force, any artificial acceleration, and if the object is intelligent in anyway, the path might change procedurally.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: impacted by **a** number

Also maybe add punctuation after acceleration. Maybe something along the lines of "...and any artifical acceleration. Furthermore, if the object..."

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 2]
```diff
@@ -1,9 +1,56 @@
...
+
+
+Tracking of airborne objects has multiple different applications.
+
+\subsection{Airports}
+Estimated thirteen thousand "bird strikes" happen each year, which is the event of a bird colliding with a vehicles, where the majority is aircrafts\cite{WikiBirdStrike}\cite{WildlifeStrikeReport}.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: colliding with a vehicle**~~s~~**

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 3]
```diff
@@ -1,9 +1,56 @@
...
+Even though 65\% are estimated to cause little damages, the damages are estimated to cost 1.2 billion dollars each year, world-wide \cite{CostOfBirdstrikes}.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: I think the first damage should be singular, but not sure.

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 4]
```diff
@@ -1,9 +1,56 @@
...
+The majority of these accidents happen at low altitude, during landing or takeoff\cite{CostOfBirdstrikes}.
+Airports utilize different methods of bird control, including killing, trapping, poison and lights\cite{BirdControlAtAirports}.
+Shooting birds is one of the most common methods, and has been observed as been quite effective, while also reinforcing other methods.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: "and has been observed to be quite effective" eller andet.

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 5]
```diff
@@ -1,9 +1,56 @@
...
+This both kills a singular bird, but also scares away other birds, however it can be quite expensive in salaries and effort.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: . However,
Eller i hvert fald bare komma efter however

Comment (technight) [reply report/1.Introduction/1.IntroductionCtrl.tex]: in both salaris and effort

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 6]
```diff
@@ -1,9 +1,56 @@
...
+Before hitting an airborne target, it is important to estimate its trajectory, and the problem can therefore be simplified to two subproblems.
+\begin{enumerate}
+	\item Calculating how many seconds it takes before a projectile can reach the target
+	\item Calculating where the target will be in T seconds
+\end{enumerate}
+Combined, these two problems means finding the spot where the travel-time of the target meets the travel-time of the projectile.
+
+This report is build on courses in both real-time systems as well as machine learning, and this problem works in the context of both. 
+In regards to machine intelligence, the problem of tracking an airborne target has potential for intelligent analysis by an artificial agent. 
+Likewise, it is a time-critical task to move a weapon so it aims at the target and hits the target, before the target is out of range or hits the ground.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: weapon so **that** it

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 7]
```diff
@@ -1,9 +1,56 @@
...
+
+As hitting the target with a projectile introduces a lot of unknown variables, this report will focus on tracking the target with a laser, as the travel time is near instant (based on the speed of light), and it's less affected by gravity. 
+The report will focus on using a ball as a target, as these are more predictable, and doesn't in anyway harm living creatures.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: Ved ikke om vi skal specificere at det er en bold. Er stadig ikke sikker på, det overhovedet skal være en bold.

**Pull request merged**